doc/ref_model/chapters/chapter03.md

```diff
@@ -41,9 +41,9 @@ The abstraction model for the NFVI makes use of the following layers (only the v
 <p align="center"><b>Figure 3-1:</b> NFVI Model Overview.</p>
 
 The functionalities of each layer are as follows:
-- **Physical Infrastructure Resources:** These consist of physical hardware components such as servers, (including random access memory, local storage, network ports, and hardware acceleration devices), storage devices, network devices, and the basic input output system (BIOS).
-- **NFVI Management Software:** This consists of both the host Operating System (OS) responsible for managing the physical infrastructure resources as well as the virtualization/containerization technology which, on request, dynamically allocates hardware components and exposes them as virtual resources.
 - **Virtual Infrastructure Resources:** These are all the infrastructure resources (compute, storage and networks) which the NFVI provides to the workloads such as VNFs/CNFs. These virtual resources can be managed by the tenants and tenant workloads directly or indirectly via an application programming interface (API).
+- **NFVI Management Software:** This consists of the software components that manage the physical and virtual resources and make those management capabilities accessible via one or more APIs. The host Operating System (OS) is responsible for managing the physical infrastructure resources and abstracting them from processes running within the OS. Virtualisation / containerisation technology dynamically allocates these abstracted hardware components and exposes them as virtual resources. Additional software is responsible for the management of logical constructs such as tenants, tenant workloads, resource catalogues, identities, access controls, security policies, etc.
```

Comment:
The sentence about "Additional software is responsible for the management of logical constructs such as tenants, etc" might be confusing. What kind of tenants, catalogues etc. is it referring to in this context?

Reply:
VIM/NFVI tenants, VM catalogues, and so on - essentially the features that you'd expect from a VIM, but that aren't included in the rest of the description for "NFVI Management Software".

Reply:
Yes. That's explained in 3.3 below. The short sentence here could be confused with the VIM which has similar things. But just 2 c.

Reply:
Yeah the purpose of this PR is to match the description here with the content in 3.3 - so we are deliberately including VIM functionality in this description here. Before this PR, they were different and had been causing confusion.